index.html

```diff
@@ -6,7 +6,170 @@
+          <img class="hero-devider" src="">
+      </div>
+
+      <section class="advantages">
+        <h2 class="visually-hidden">Преимущества</h2>
+        <section class="advantages-short-description">
+          <h2 class="advantages-title">Седона — небольшой городок в Аризоне, заслуживающий большего!</h2>
+          <p class="advantages-description">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+        </section>
+        <ul class="advantages-list">
+          <li class="advantages-item advantages-item-main">
+            <h3 class="advantages-item-main-title">Настоящий городок</h3>
+            <p class="advantages-item-main-rating">— №1 —</p>
+            <p class="advantages-item-main-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+            <img class="advantages-item-img" src="" alt="На переднем плане кусты и здание, на заднем плане горы">
+            <ul class="advantages-list-item">
+              <li class="advantages-item-normal">
+                <img class="housing" src="">
+                <h3 class="advantages-normal-title">Жильё</h3>
+                <p class="advantages-normal-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+              </li>
+              <li class="advantages-item">
+                <img class="meal" src="">
+                <h3 class="advantages-title">Еда</h3>
+                <p class="advantages-description">Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+              </li>
+              <li class="advantages-item-normal">
+                <img class="souvenirs" src="">
+                <h3 class="advantages-normal-title">Сувениры</h3>
+                <p class="advantages-normal-description">Не только китайского, но и настоящего местного производства!</p>
+              </li>
+            </ul>
+          </li>          
+
+          <li class="advantages-item advantages-item-main">
+            <img class="advantages-item-img" src="" alt="Пейзаж с горами">
+            <h3 class="advantages-item-main-title">Там есть мост дьявола</h3>
+            <p class="advantages-item-main-rating">— №2 —</p>
+            <p class="advantages-item-main-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+          </li>
+
+          <li class="advantages-item-light">
+            <h3 class="advantages-item-light-title">Небольшая площадь</h3>
+            <p class="advantages-item-light-rating">— №3 —</p>
+            <p class="advantages-item-light-description">Все достопримечательности находятся очень близко</p>
+          </li>
+          <li class="advantages-item-normal">
+            <h3 class="advantages-item-normal-title">Красивая дорога</h3>
+            <p class="advantages-item-normal-rating">— №4 —</p>
+            <p class="advantages-item-normal-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+          </li>
+          <li class="advantages-item-light">
```

Comment:
```suggestion
          <li class="advantages-item advantages-item-light">
```
Я бы добавил общий класс, т.к. эти преимущества (№3 - №5) отличаются от первого и второго только фоном и отсутствием фоток и деталей. Поэтому класс у них будет общий, а `advantages-item-light` будет просто менять фон

Reply:
исправил